packit/cli/build.py

```diff
@@ -43,14 +43,17 @@
     help="Path to dist-git repo to work in. "
     "Otherwise clone the repo in a temporary directory.",
 )
+@click.option("--koji-target", help="Koji target to build inside.")
```

Comment:
Could this also say: `"(see 'koji list-targets')"` ?

Reply:
nykdi